README.md

```diff
@@ -20,3 +20,75 @@ with a few commands. Need an ingress to serve traffic? Use Juju to deploy and
 integrate a range of ingresses, such as
 [Traefik](https://charmhub.io/traefik-k8s), and expose your application to
 external traffic in seconds.
+
+## Contributing
+
+Is there something missing, for example, a framework that you would like to use?
```

Comment:
I find this sentence formulation a little awkward. Possibly "Is something missing" could be "Is there something missing from the PaaS App Charmer framework"?

Reply:
Thanks for the thorough review! I have implemented all of these